### ISSUE TITLE
Rendering result is gong after switching between 'stereo' and 'fullscreen'

### ISSUE THREAD

**Issue by PennyQ** [maartenbreddels/ipyvolume#19]
I basically only changed the dataset from the example code, and here is what happened after toggle between view modes:

![image](https://cloud.githubusercontent.com/assets/13411839/24160657/d33d2608-0e38-11e7-8891-8b40ceb79e5b.png)

Here is my Jupyter Notebook info:
```
The version of the notebook server is 4.4.1 and is running on:
Python 2.7.12 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:43:17) 
[GCC 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 2336.11.00)]
```
Any ideas? Thanks!

**Comment by maartenbreddels** [comment]
Hi Penny,

Judging from the black background you are using the 0.2 version, could you try upgrading to 0.3, and see if you still get the same? It was a bug, although I didn't fully understand it, I managed to get rid of it in 0.3 I believe.

cheers,

Maarten

**Comment by PennyQ** [comment]
@maartenbreddels Thanks for reply! The bug is still there after I upgraded to 0.3 using `conda install -c conda-forge ipyvolume`

And, I found the volume which was shown in nbviewer before is also gone, as seen on http://nbviewer.jupyter.org/github/PennyQ/ipython_scripts/blob/master/ipyvolume_l1448co.ipynb
I suspected these two issues are related?

**Comment by maartenbreddels** [comment]
That is odd, so you now have black background? Can you describe exactly when it happens? Only when you switch to fullscreen, or also stereo, or only when you go fullscreen while stereo mode?
Btw, I can see the volume on nbviewer,

**Comment by PennyQ** [comment]
Yes I have white background now, and the problem happened when running ipython notebook locally, after switching from stereo -> fullscreen -> quit fullscreen(press ESC), then the volume is gone as below (such problem doesn't happen on nbviewer for me), unless rerun the code:
(I am using `quickvolshow` btw)
![image](https://cloud.githubusercontent.com/assets/13411839/24179749/64da473c-0e87-11e7-8d05-ec2aeccf3af9.png)

Yes the volume in nbviewer is back now, after I updated the ipnb file with ipyvolume 3.0! Thanks for this!

**Comment by maartenbreddels** [comment]
Thanks for reporting this, I'll try to fix this for the next release!

**Comment by maartenbreddels** [comment]
Hi Penny, this is fixed now in version 0.3.2.
